web/src/main/java/org/testeditor/fixture/web/WebDriverFixture.java

```diff
@@ -194,6 +216,45 @@ public String readValue(String elementLocator) {
 		WebElement element = getWebElement(elementLocator);
 		return element.getText();
 	}
+	
+	@FixtureMethod
+	public void selectElementInSelection(String elementLocator, String value) throws InterruptedException{
+		clickOn(elementLocator);
+		Thread.sleep(300);
+		WebElement element = getWebElement(elementLocator);
+	    new Select(element).selectByVisibleText(value);
+		//logger.trace("Selected value {} in selection {}", value, elementLocator);
+	}
+	
+	@FixtureMethod
+	public Map<String, String>  getOptionsInSelection(String elementLocator) throws InterruptedException {
+		clickOn(elementLocator);
+		Thread.sleep(300);
+		Map<String, String> namesOfAllSelectedOptions = new HashMap<String, String>();
+		Select selection = new Select(getWebElement(elementLocator));
+		allSelectedOptions = selection.getAllSelectedOptions();
```

Comment:
I don't see any reason to have allSelectedOptions as an object memeber. Method scope shuld be enough.